shoppingcart/shopping_cart.py

```diff
@@ -5,8 +5,9 @@
 
 from cloudstate.cloudstate import CloudState
 from shoppingcart.shopping_cart_entity import entity as shopping_cart_entity
-import logging
 
 if __name__ == '__main__':
-    logging.basicConfig()
-    CloudState().register_event_sourced_entity(shopping_cart_entity).start()
+    CloudState()\
+        .port('8080')\
```

Comment:
do you set the port here explicitly again even if there is a default?

Reply:
shopping_cart.py is an example, it only demonstrates certain characteristics that the user can customize if necessary, internally the application has the same behavior that it had previously. Search for an environment variable and if you don't find the default arrow.